kv/txn_coord_sender.go

```diff
@@ -184,16 +187,16 @@ func (tm *txnMetadata) close(txn *proto.Transaction, resolved []proto.Key, sende
 		// We don't care about the reply channel; these are best
 		// effort. We simply fire and forget, each in its own goroutine.
 		if stopper.StartTask() {
-			go func() {
+			go func(ctx context.Context) {
```

Comment:
Is there any advantage to adding parameters to the immediately invoked function, rather than just using them as a closure?